index.html

```diff
@@ -141,28 +141,27 @@ <h2>Glossary</h2>
 
 <div class="letter_anchor" id="a">A</div>
 
-<p><dfn data-plurals="application internal identifiers" id="dfn-application-internal-identifier" class="lint-ignore export">Application internal identifiers</dfn>. Identifiers defined by or assigned by a user in a <a href="#def_vocabulary" class="termref">vocabulary</a> that is internal to the document format or protocol and not intended for human interaction. Such values are generally not <a href="#def_localizable_text" class="termref">localizable text</a>.</p>
-
-<p><dfn data-lt="ASCII case-insensitive|ASCII case-insensitive matching" id="def_ascii_case-insensitive" class="lint-ignore export">ASCII case-insensitive matching</dfn>. <a href="https://infra.spec.whatwg.org/#ascii-case-insensitive">Defined</a> in [[INFRA]], this compares two sequences of code points as if all ASCII code points in the range 0x41 to 0x5A (A to Z) were mapped to the corresponding code points in the range 0x61 to 0x7A (a to z), but other code points are not <a class="termref" href="#def_case_folding">case-folded</a>. ASCII case-insensitive matching can be required when a <a href="#def_vocabulary" class="termref">vocabulary</a> is itself constrained to ASCII.</p>
-
-
+<p><dfn class="lint-ignore export">Abjad</dfn>. A writing system in which consonants are indicated, but not short vowels. The term “abjad” is derived from the first four letters of the traditional order of the Arabic script: <i class="name">alef, beh, jeem, dal</i>. (See also the <a href="https://www.unicode.org/glossary/#abjad">Unicode definition</a> and <a href="http://www.unicode.org/versions/latest/ch06.pdf#G7382">Section 6.1, Writing Systems</a>.)</p>
```

Comment:
why?

Reply:
The renderer does a good enough job putting appropriately shaped ones in and it might be easier for translators. Not critical, hence the question mark.